extensions-core/groovy-dsl/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,31 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This is a generated file. Do not edit directly!
+# To re-generate, run the following command from the top level directory:
+#
+#   mvn -N cq:update-quarkus-metadata
+#
+---
+name: "Camel Groovy DSL"
+description: "Support for parsing Groovy route definitions at runtime"
+metadata:
+  guide: "https://camel.apache.org/camel-quarkus/latest/reference/extensions/groovy-dsl.html"
+  categories:
+  - "integration"
+  status:
+  - "stable"
```

Comment:
@ppalaga how can I mark this extension as somehow "experimental"  ?

Reply:
This will also be auto-solved by moving to extensions-jvm. All extensions there are marked as experimental by default.